demisto_sdk/commands/unify/unifier.py

```diff
@@ -443,7 +444,9 @@ def add_contributors_support(self, unified_yml: Dict, contributor_type: str, con
         else:
             contributor_description = CONTRIBUTOR_DETAILED_DESC.format(contributor_type.capitalize(), author)
             if contributor_email:
-                contributor_description += f'\n- **Email**: [{contributor_email}](mailto:{contributor_email})'
+                email_list: List[str] = contributor_email if isinstance(contributor_email, list) else [contributor_email]
+                for email in email_list:
+                    contributor_description += f'\n- **Email**: [{email}](mailto:{email})'
```

Comment:
This seems wrong. `contributor_email` is a string even if the metadata contains a list of emails seems like we need to split here...

Reply:
spoke with you on slack. 
we receive emails from json file, therefore we will receive a list of strings or one string.

Reply:
Not necessarily true, as I said in slack we need to support both (lists or lists of strings) something like argToList - I think we already have a function in SDK tools for it. FYI in the metadata given in the issue example, the emails were a string

Reply:
@ShahafBenYakir - not sure we have that in SDK tools but could be a good addition.

Reply:
added function